### ISSUE TITLE
Accessibility issues with BLM landing page

### ISSUE THREAD

**Issue by MylesBorins** [nodejs/nodejs.org#3257]
Hey All,

We received an email report that the links on the BLM landing page were hard to read for those who can't distinguish color contrast well.

Did a quick report and saw that there are indeed some accessibility improvements we can make on the page

https://try.powermapper.com/demo/Report/577ee505-f5f6-425e-b1ca-450447bd7cce

I've not run this across the entire site, but as the BLM landing page borrows a bunch of the broader styles it might be a site wide issue. i'll try and find some time to fix this, but if anyone wants to take a pass at it first please don't wait for me.

**Comment by davidfekke** [comment]
A lot of organizations have a banner and link for supporting the BLM movement. It might be a better UX experience to move the BLM support to another page, and have a link to it on the homepage.

I know I came to the nodejs.org site the other day to try to download the installer, and I had to search for awhile until I found it at the bottom of the page.

**Comment by nschonni** [comment]
@davidfekke that already happened in #3246

@MylesBorins Looking at Wave, I'm not seeing any contrast issues flagged https://wave.webaim.org/report#/https://nodejs.org/en/black-lives-matter/ but that can sometimes be subjective, even if the ratio is correct

**Comment by MylesBorins** [comment]
@davidfekke the change to make a banner landed yesterday. The original plan was to replace the landing page for a week.

**Comment by Trott** [comment]
> @MylesBorins Looking at Wave, I'm not seeing any contrast issues flagged https://wave.webaim.org/report#/https://nodejs.org/en/black-lives-matter/ but that can sometimes be subjective, even if the ratio is correct

https://wave.webaim.org/report#/https://nodejs.org/en/black-lives-matter/ flags 32 color contrast issues for me, all in links. 

<img width="523" alt="Screen Shot 2020-06-16 at 9 20 58 PM" src="https://user-images.githubusercontent.com/718899/84854758-4c5b2700-b017-11ea-8557-2befb8b70075.png">

**Comment by nschonni** [comment]
Weird, maybe a browser related thing? I'm running in Firefox on Windows 10
![image](https://user-images.githubusercontent.com/1297909/84854902-c946ca80-b030-11ea-86d1-056a1360eb1c.png)

**Comment by Trott** [comment]
> Weird, maybe a browser related thing? I'm running in Firefox

I was running in Chrome but I get the same 32 errors in Firefox too. I'm on macOS.

**Comment by nschonni** [comment]
OK, think I figured out why I wasn't seeing it. I use Stylish for custom CSS, and even though I toggled it off, it seems like it was still picked up in the iframe

**Comment by XhmikosR** [comment]
The main page passes WCAG2AA AFAICT. I remember I checked it in the past also.

EDIT: NVM, this issue is for https://nodejs.org/en/black-lives-matter/ which indeed has issues.
